dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpServlet.java

```diff
@@ -114,7 +114,7 @@ private void callTool(McpTransport transport) {
         Object bean = bm.getReference(params.getBean(), params.getBean().getBeanClass(), cc);
         try {
             Object result = params.getMethod().invoke(bean, params.getArguments(jsonb));
-            transport.sendResponse(new ToolResponseResult(result, false));
+            transport.sendResponse(new ToolResponseResult(result.toString(), false));
```

Comment:
```suggestion
            transport.sendResponse(new ToolResponseResult(Objects.toString(result), false));
```

Avoids possible null pointer exception.